packages/npm/todo/src/operation-handlers/todo.ts

```diff
@@ -1,24 +1,18 @@
-import * as api from 'todo-api';
+import * as api from "todo-api";
 import { CommandHandlers } from "../commandHandlers/commandHandlers.js";
 import CreateTodo from "../commands/createTodo.js";
 
-
-type TodoItem = {
-  description: string;
-  id: number;
-  done: boolean;
-};
-
-export const addTodoOperationHandler: api.server.AddTodoItemOperationHandler<{}> = async (todo: any) => {
+export const addTodoOperationHandler: api.server.AddTodoItemOperationHandler<{}> = async (todo) => {
   const createTodoCommand = new CreateTodo(todo.description);
   const commandHandlers = new CommandHandlers();
 
   const createdTodo = await commandHandlers.createTodo(createTodoCommand);
 
-  const todoItem: TodoItem = {
+  const todoItem = {
     description: createdTodo.todoName,
     id: createdTodo.todoId,
     done: createdTodo.todoIsDone,
   };
+
   return todoItem;
```

Comment:
If the structure of todoItem does not match, then the signature of the function does not match. This is because we return todoItem. So if this is the case then we get an error!